### PR TITLE
refactor: use MobileStack references in ios build

### PR DESCRIPTION
**Pull request by kathaypacific**
### Description

Related to making the iOS build brand agnostic. Main changes here:
- rename `celo` -> `MobileStack`
- remove unused CeloTest and Celo-tvOS targets
- since the xcode workspace is renamed, also update any scripts and workflows that rely on the path

### Test plan

Tested the iOS build locally + CI.

### Related issues

- Related to RET-1157

### Backwards compatibility

Y

### Network scalability

If a new NetworkId and/or Network are added in the future, the changes in this PR will:

- [ ] Continue to work without code changes, OR trigger a compilation error (guaranteeing we find it when a new network is added)

### CODE REVIEW

File: ios/MobileStack/AppDelegate.mm [hunk 1]
```diff
@@ -16,7 +16,8 @@
 #import <React/RCTLinkingManager.h>
 #import <React/RCTHTTPRequestHandler.h>
 
-@import Firebase;
+#import <FirebaseCore/FirebaseCore.h>
+#import <FirebaseAuth/FirebaseAuth.h>
```
Comment (kathaypacific) [inline ios/MobileStack/AppDelegate.mm:20]: i don't know why `@import Firebase;` stopped working, but it gave me build errors that seem unrelated to my changes here. the build error was:
```
AppDelegate.mm:19:1: error: use of '@import' when C++ modules are disabled, consider using -fmodules and -fcxx-modules
@import Firebase;
^
/Users/user/Documents/Workspace/wallet/ios/MobileStack/AppDelegate.mm:83:6: error: use of undeclared identifier 'FIROptions'; did you mean 'kNilOptions'?
    [FIROptions defaultOptions].deepLinkURLScheme = deepLinkUrlScheme;
     ^~~~~~~~~~
     kNilOptions
In file included from /Users/user/Documents/Workspace/wallet/ios/MobileStack/AppDelegate.mm:10:
In file included from /Users/user/Documents/Workspace/wallet/ios/MobileStack/AppDelegate.h:10:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator17.5.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIKit.h:8:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator17.5.sdk/System/Library/Frameworks/Foundation.framework/Headers/Foundation.h:6:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator17.5.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:54:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator17.5.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CFBase.h:80:
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator17.5.sdk/usr/include/MacTypes.h:393:3: note: 'kNilOptions' declared here
  kNilOptions                   = 0
  ^
/Users/user/Documents/Workspace/wallet/ios/MobileStack/AppDelegate.mm:83:6: error: bad receiver type '(unnamed enum at /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator17.5.sdk/usr/include/MacTypes.h:392:1)'
    [FIROptions defaultOptions].deepLinkURLScheme = deepLinkUrlScheme;
     ^~~~~~~~~~
/Users/user/Documents/Workspace/wallet/ios/MobileStack/AppDelegate.mm:84:6: error: use of undeclared identifier 'FIRApp'
    [FIRApp configure];

```

File: ios/MobileStackTests/MobileStackTests.m
Comment (jeanregisser) [inline ios/MobileStackTests/MobileStackTests.m:1]: We can delete this folder too.

**Review by jeanregisser — APPROVED** [review]
🚀
